tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/CreateCatalogueSolutionOrderItemSteps.cs

```diff
@@ -38,13 +33,10 @@ public void GivenTheUserCreatesARequestToAddANewCatalogueSolutionOrderItemToTheO
                 orderId);
         }
 
-        [Given(@"the user enters '(.*)' payload")]
+        [Given(@"the user enters the '(.*)' create catalogue solution order item request payload")]
```

Comment:
Glad to see this change! Was looking at this in dev this morning and reached the same conclusion you seem to have, here